src/legacy/core_plugins/kibana/public/table_list_view/table_list_view.js

```diff
@@ -307,6 +309,53 @@ class TableListViewUi extends React.Component {
     );
   }
 
+  columns() {
+    const canEdit = this.state.items.some(item => item.canEdit);
+    const originalColumns = this.props.tableColumns;
+
+    if (!canEdit) {
+      return originalColumns;
+    }
+
+    return [
+      ...originalColumns,
+      {
+        field: 'canEdit',
+        align: 'right',
+        width: '100px',
+        name: i18n.translate('kbn.table_list_view.listing.table.actionTitle', {
+          defaultMessage: 'Actions'
+        }),
+
+        render: (_field, record) => (
+          <EuiButtonIcon
```

Comment:
tooltip is missing here - the eui implementation is here: https://github.com/elastic/eui/blob/master/src/components/basic_table/default_item_action.js